### PR TITLE
hale moshkele etebar

### PR DESCRIPTION
**Pull request by LordST95**

### CODE REVIEW

File: accounts/templates/header.html [hunk 1]
```diff
@@ -161,17 +31,16 @@
...
-    {#                            <li><a href="#"><i class="icon-cog"></i>تنظیمات</a></li>#}
-    {#                            <li><a href="#"><i class="icon-bell-alt"></i>اعلام ها</a></li>#}
                                 <li><a href="{% url 'registration_logout' %}"><i class="icon-key"></i>خروج</a></li>
                             </ul>
                         </li>
 
                         <li class="">
                             <a class="" href="#">
                                 <span class="username">اعتبار حساب: </span>
-                                <span class="username">{{ user.credit_card.asset }}</span>
```
Comment (mh-izadi) [inline accounts/templates/header.html]: credit_card_set nabayad mizashti?

Comment (LordST95) [reply accounts/templates/header.html]: Nmidoonam bezar ie test konam

Comment (LordST95) [reply accounts/templates/header.html]: na j Nmide
hamine

Comment (mh-izadi) [reply accounts/templates/header.html]: are chon khodet related name ro set kardi hamin mishe agar nakarde bodi defaultesh hamin ke goftam mishod

**Pull request merged**